### PR TITLE
Support for changes to Okta Login Flow

### PR DESCRIPTION
**Pull request by bromanko**
- Changes to selectors 
- Changes to how we extract the SAMLResponse to deal with redirects that were faster than we could detect

### CODE REVIEW

File: src/providers/okta.js [hunk 1]
```diff
@@ -26,20 +26,43 @@ const Okta = {
...
+        // a hidden input with the SAML response data. Javascript then immediately submits the
+        // form once the page loads. This means there is limited time to extract the SAML response.
+        // Each `wait` call is capable of detecting the SAML response. However, then executing an
+        // `evaluate` call takes long enough for the SAML response to no longer be available.
+        // So, we need to capture the SAML response in the `wait` call. The easiest mechanism to
+        // get data from the browser back to Node is via the console logger. This event handler
+        // looks for a log including the SAML response.
+        //
+        // This event handler is so early in the chain because it must be attached prior to calling
+        // `goto`.
```
Comment (jsatk) [inline src/providers/okta.js]: Thank you for this comment.

File: src/providers/okta-helpers.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  waitAndEmitSAMLResponse: function () {
+    /* eslint-disable  */
+
+    // Stop waiting if an error was found
+    var errEl = document.querySelector('.o-form-has-errors');
+    if (errEl) return true;
+
+    // Keep waiting if no error or SAMLResponse detected
+    var samlEl = document.querySelector('input[name="SAMLResponse"]');
+    if (!samlEl) return false;
```
Comment (warhod) [inline src/providers/okta-helpers.js]: I suppose the calling `wait()` function will handle the retry until the default 30 seconds timeout?

File: src/providers/okta.js [hunk 3]
```diff
@@ -26,20 +26,43 @@ const Okta = {
 
     spinner.start();
 
-    let nightmare = Nightmare({show: process.env.DEBUG});
+    let samlAssertion = undefined;
+    let nightmare = Nightmare({
+      show: process.env.DEBUG,
+      openDevTools: true,
+      typeInterval: 20,
```
Comment (warhod) [inline src/providers/okta.js]: I tried it with 5ms and my hyper-sensitive brain can tell that it's faster... ;-)

**Comment by warhod** [comment]
Awesome Sauce

File: src/providers/okta-helpers.js (hunk 2 above)
Comment (bromanko) [inline src/providers/okta-helpers.js]: Will add `waitTimeout: 30 * 1000`

File: src/providers/okta.js (hunk 3 above)
Comment (bromanko) [inline src/providers/okta.js]: I will reduce it to 5

**Pull request merged**